.github/workflows/javascript-formatting.yml

```diff
@@ -32,41 +31,38 @@ jobs:
           repository: ${{ github.event.pull_request.head.repo.full_name }}
           fetch-depth: 0
 
-      - name: Prettify code
-        uses: creyD/prettier_action@v4.3
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v37
         with:
-          # This part is also where you can pass other options, for example:
-          prettier_options: --write **/*.{js,ts,jsx,tsx,css,md}
-          only_changed: true
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-  linter:
-    name: Javascript linter
-    runs-on: ubuntu-latest
-    defaults:
-      run:
-        working-directory: ./web
-
-    steps:
-      - id: skip_check
-        uses: fkirc/skip-duplicate-actions@v5
-        with:
-          concurrent_skipping: 'same_content_newer'
-
-      - name: Checkout
-        uses: actions/checkout@v3
-        with:
-          # Make sure the actual branch is checked out when running on pull requests
-          ref: ${{ github.event.pull_request.head.ref }}
-          repository: ${{ github.event.pull_request.head.repo.full_name }}
-          fetch-depth: 0
+          files_yaml: |
+            src:
+              - '**.js'
+              - '**.ts'
+              - '**.tsx'
+              - '**.jsx'
+              - '**.css'
+              - '**.md'
 
       - name: Install Dependencies
         run: npm install
 
       - name: Lint
-        run: npm run lint
+        if: steps.changed-files.outputs.any_changed == 'true'
```

Comment:
Hi @gabek, I'm the author of [tj-actions/changed-files](https://github.com/tj-actions/changed-files) and I wanted to draw your attention to a bug with this workflow.

Using the `files_yaml` input introduces outputs namespaced by the key you provide in this case `src`.

You'll need to change this if condition to use the same key provided in order to prevent hidden issues.


``` suggestion
if: steps.changed-files.outputs.src_any_changed == 'true'
```